src/configuration/customers/customer-configuration.md

```diff
@@ -77,26 +77,49 @@ _Account Information Options_
 |Change Email Template|Store View|Identifies the default email template that is used when a customer changes their email address.|
 |Change Email and Password Template|Store View|Identifies the default email template that is used when a customer changes their email address and password.|
 
+<!--{% if "Default.CE Only" contains site.edition %}-->
+
 ## Name and Address Options
 
-<!--{% if "Default.CE Only" contains site.edition %}-->
-![]({% link images/images/config-customers-customer-configuration-name-address.png %}){: .zoom}
+![]({% link images/images/config-customers-customer-configuration-name-address-options.png %}){: .zoom}
 [_Name and Address Options_]({% link customers/name-address-options.md %})
+
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|Number of Lines in a Street Address|Website|Determines the number of lines in the street address. The street address consists of one to four lines. If the field is blank, the default street address of two lines is used.|
```

Comment:
in step 2 of the name and address topic, it states the default lines is 3.  is this the same?

Enter a number from 1 to 4 for the **Number of Lines in a Street Address**. By default, the street address is 3 lines.

Reply:
Fixed.